test/parallel/test-http-connect.js

```diff
@@ -62,6 +62,7 @@ server.listen(0, common.mustCall(() => {
     assert(!socket.onend);
     assert.strictEqual(socket.listeners('connect').length, 0);
     assert.strictEqual(socket.listeners('data').length, 0);
+    assert.strictEqual(socket.listenerCount('drain'), 0);
```

Comment:
I guess these could be kept consistent if we want to?

Reply:
Sure, I actually wanted to change all of them to `listenerCount` but that's a perfect code+learn exercise, I guess.